sdk/src/wallet/account/operations/participation/voting_power.rs

```diff
@@ -169,7 +168,6 @@ where
 
             participation_bytes
         } else {
-            // TODO participation bytes are incorrect, should we really just ignore?
```

Comment:
I would ignore, otherwise we could get in a state where users can't do anything with their wallet anymore